include/libtorrent/settings_pack.hpp

```diff
@@ -792,6 +792,9 @@ namespace libtorrent
 			// time to wait until a new retry of a web seed takes place
 			urlseed_wait_retry,
 
+			// if unavailable, retry in these minutes
+			web_seed_name_lookup_retry_minutes,
```

Comment:
please add a little bit more comprehensive documentation for this setting. I also believe all (afaict) times are specified in seconds. I think it would make sense to specify this in seconds as well.